macros/base/end_print.cfg

```diff
@@ -17,6 +16,7 @@ gcode:
     PARK
 
     {% if klippain_mmu_enabled %}
+        {% set mmu_unload = params.MMU_UNLOAD_AT_END|default(printer["gcode_macro _USER_VARIABLES"].mmu_unload_on_end_print)|int %}
```

Comment:
and remove this one

Reply:
why remove it? does that mean we don't use MMU_UNLOAD_AT_END parameter at all? Then we can remove all references of it in Klippain